src/features/bakery/components/CraftingItems.tsx

```diff
@@ -0,0 +1,132 @@
+      item: selected.name,
+      amount,
+    });
+    setToast({ content: "SFL -$" + selected.price.mul(amount) });
+    selected.ingredients.map((ingredient, index) => {
+      setToast({
+        content:
+          "Item " + ingredient.item + " -" + ingredient.amount.mul(amount),
+      });
+    });
+
+    shortcutItem(selected.name);
+  };
+
+  return (
+    <div className="flex">
+      <div className="w-3/5 flex flex-wrap h-fit">
+        {Object.values(items).map((item) => (
+          <Box
+            isSelected={selected.name === item.name}
+            key={item.name}
+            onClick={() => setSelected(item)}
+            image={ITEM_DETAILS[item.name].image}
+            count={inventory[item.name]}
+          />
+        ))}
+      </div>
+      <OuterPanel className="flex-1 w-1/3">
+        <div className="flex flex-col justify-center items-center p-2 relative">
+          <span className="text-shadow text-center">{selected.name}</span>
+          <img
+            src={ITEM_DETAILS[selected.name].image}
+            className="h-16 img-highlight mt-1"
+            alt={selected.name}
+          />
+          <span className="text-shadow text-center mt-2 sm:text-sm">
+            {selected.description}
+          </span>
+
+          {!hasSelectedFood && 
+            <div className="border-t border-white w-full mt-2 pt-1">
+              {selected.ingredients.map((ingredient, index) => {
+                const item = ITEM_DETAILS[ingredient.item];
+                const lessIngredient = new Decimal(
+                  inventory[ingredient.item] || 0
+                ).lessThan(ingredient.amount);
+
+                return (
+                  <div className="flex justify-center items-end" key={index}>
+                    <img src={item.image} className="h-5 me-2" />
+                    <span
+                      className={classNames(
+                        "text-xs text-shadow text-center mt-2 ",
+                        {
+                          "text-red-500": lessIngredient,
+                        }
+                      )}
+                    >
+                      {ingredient.amount.toNumber()}
+                    </span>
+                  </div>
+                );
+              })}
+
+              <div className="flex justify-center items-end">
+                <img src={token} className="h-5 mr-1" />
+                <span
+                  className={classNames("text-xs text-shadow text-center mt-2 ", {
+                    "text-red-500": lessFunds(),
+                  })}
+                >
+                  {`$${selected.price.toNumber()}`}
+                </span>
+              </div>
+            </div>
+          }
+          <Button
+            disabled={hasSelectedFood ? false : !canCraft}
+            className={`${hasSelectedFood ? "pe-none" : ""} text-xs mt-1`}
+            onClick={() => craft()}
+          >
+            {hasSelectedFood ? "Already crafted" : "Craft"}
```

Comment:
I would suggest not putting the "Already crafted" text inside of the button. When players see a button, they will still try to click it.

Consider showing just plain text here.